app/api/v1/endpoints/tenants.py

```
@@ -1,26 +1,28 @@
-    DbOrFirestore,
+    ensure_audit_logged,
     get_tenant_creation_service,
     get_tenant_id,
     get_tenant_repo,
     get_tenant_repo_for_write,
     get_verified_tenant_id,
     require_permission,
-    _get_db_or_firestore_write,
 )
+from app.infrastructure.persistence.database import get_db_transactional
 from app.application.dtos.user import UserResult
```

Flake8 (7.3.0)

[error] 'app.application.dtos.user.UserResult' imported but unused

(F401)